doc/man1/openssl-cmp.pod.in

```diff
@@ -601,27 +608,32 @@ is typically used when authenticating with pre-shared key (password-based MAC).
 
 =item B<-secret> I<arg>
 
-Source of secret value to use for creating PBM-based protection of outgoing
-messages and for verifying any PBM-based protection of incoming messages.
+Prefer PBM-based message protection with given source of secret value.
+This useed for creating PBM-based protection of outgoing messages and as far
```

Comment:
This useed for ??

Reply:
Fixed this typo and further improved the sentence.